<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlanViewer</class>
 <widget class="QWidget" name="PlanViewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>731</width>
    <height>524</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTextEdit" name="textViewer">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>10</y>
     <width>481</width>
     <height>491</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="newPlanButton">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>60</y>
     <width>88</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>newplan</string>
   </property>
  </widget>
  <widget class="QPushButton" name="planDataButton">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>100</y>
     <width>88</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>plandata</string>
   </property>
  </widget>
  <widget class="QPushButton" name="planReviewButton">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>140</y>
     <width>88</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>planreview</string>
   </property>
  </widget>
  <widget class="QPushButton" name="setStateButton">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>180</y>
     <width>88</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>setState</string>
   </property>
  </widget>
  <widget class="QPushButton" name="newExecutiveButton">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>220</y>
     <width>88</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>newExecutive</string>
   </property>
  </widget>
  <widget class="QPushButton" name="getPlansButton">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>260</y>
     <width>88</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>getPlans</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
